manifests/init.pp

```diff
@@ -95,6 +95,10 @@
 #   (string) HTTP index server to use for pip/virtualenv.
 #   Defaults to false ($::puppetboard::params::python_index)
 #
+# [*python_use_epel*]
+#   (bool) Whether the Python class will use attempt to manage EPEL or not.
+#   Defaults to true ($::puppetboard::params::python_use_epel)
+#
```

Comment:
The default for fedora users was actually `false` though. https://github.com/stankevich/puppet-python/blob/718e9bc482f72559d9035ec2b8b651be01984128/manifests/params.pp#L25  So this would change the default behaviour for them.

I'm not sure we want to be replicating the logic in that class in our own params.pp though...  Dunno.  Maybe someone else will have a suggestion.